partials/model-class.js

```diff
@@ -0,0 +1,52 @@
+const _ = require('lodash');
```

Comment:
instead of partials people should rather opt in to use `components` reference for reusable components - more natively common word in the context of react and react components concept